protos/flyteidl/admin/node_execution.proto

```diff
@@ -35,6 +35,10 @@ message NodeExecutionListRequest {
     // Sort ordering.
     // +optional
     Sort sort_by    = 5;
+
+    // Unique identifier of the parent node in the execution
+    // +optional
+    string unique_parent_id = 6;
```

Comment:
why is this a string and not NodeExecutionIdentifier?

Reply:
Nope. I did not want that. Why duplicating everything, when we know that nodes cannot be across execution?

Reply:
I don't see the endpoint definition in this changeset, so I'm not sure how this new field will be used when querying?
I guess I'm wondering why we are choosing to have it as a separate field rather than a filter?

Reply:
True. Removing it. This will be just another filter.

Reply:
We discussed this offline and decided a separate endoint/request parameter is more appropriate than a filter, since specifying this value will return the base set of results and not filter down the set of results that would be returned if you left the value empty.